e2e/tests/apikey-lifecycle.spec.ts

```
@@ -349,3 +357,136 @@ test.describe('API Key Lifecycle', () => {
+      // Step 2: approve modal shows expiration date
+      await expect(row.locator('[aria-label="Actions"]')).toBeEnabled({ timeout: 30_000 });
+      await row.locator('[aria-label="Actions"]').click();
+      await page.locator('[role="menuitem"]:has-text("Approve")').click();
+      const modal = page.locator('.pf-v6-c-modal-box');
+      await expect(modal.locator('text=Aug 27, 2027')).toBeVisible({ timeout: 10_000 });
+
+      // Step 3: approve
+      await modal.locator('button:has-text("Approve")').click();
+      await expect(
+        page.locator('.pf-v6-c-alert:has-text("approved successfully")'),
+      ).toBeVisible({ timeout: 30_000 });
```

ESLint

[error] Replace `⏎········page.locator('.pf-v6-c-alert:has-text("approved·successfully")'),⏎······).toBeVisible({·timeout:·30_000` with `page.locator('.pf-v6-c-alert:has-text("approved·successfully")')).toBeVisible({⏎········timeout:·30_000,⏎·····`

(prettier/prettier)